### DISCUSSION TITLE
How would you return an SVG string from a function/type?

### DISCUSSION THREAD

**Discussion by radianrad** [josdejong/mathjs#3109] [Q&A]
I'm trying to implement a few functions that return custom SVGs that will be rendered in the output.

I get pretty close, but my guess is the parser is messing with the SVG and I need a different way. I tried a custom HTML handler, but I think that only works for LaTex and strings.

Is there a way to override toHTML for a dataType or is there a simpler method other than pre-filtering expressions for a specific node and modifying the evaluator function?

Thank you!

**Comment by josdejong** [comment]
Good point, the custom hander should work with the `onHTML` method too, but it didn't. I've fixed this via a6cc8e71289e80ee8acb6b74b81a1977fd16f984, and published in `mathjs@12.2.1`.

**Reply by radianrad** [reply]
Thank you for the speedy fix!
